### DISCUSSION TITLE
如何搭建博客站点 | 右耳朵猫(whinc)

### DISCUSSION THREAD

**Discussion by giscus** [whinc/blog#32] [Announcements]
# 如何搭建博客站点 | 右耳朵猫(whinc)

本文主要记录个人站点 whinc.github.io 的搭建过程，以及后续的维护更新，可以看做是个人站点自身的一份 changelog

http://localhost:3000/blog/2023/how-to-setup-blog

**Comment by whinc** [comment]
测试
